example/package.json

```diff
@@ -8,12 +8,13 @@
     "client": "NODE_OPTIONS=--openssl-legacy-provider expo start --dev-client",
     "web": "NODE_OPTIONS=--openssl-legacy-provider expo start --web",
     "web:export": "NODE_OPTIONS=--openssl-legacy-provider expo export:web",
-    "android": "expo run:android",
-    "ios": "run-s pod-install run:ios",
+    "android": "run-s check:prebuild run:android",
+    "run:android": "expo run:android",
+    "ios": "run-s check:prebuild run:ios",
     "run:ios": "expo run:ios",
-    "pod-install": "pod-install",
```

Comment:
The dependency can be removed as well

Reply:
Fixed in [fe7926a](https://github.com/react-navigation/react-navigation/pull/11020/commits/fe7926a8981836ae2fe6ed6d08f2531b8f3b5dc5)